### PR TITLE
Add link to plugin developers guide from README

### PR DESCRIPTION
**Pull request by alb-i986**
It is now clear for plugin developers willing to support JCaSC how to go about it from the README.
See also https://issues.jenkins-ci.org/browse/JENKINS-62565

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,6 +244,10 @@ Kubernetes users:\
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=18341) for known compatibility issues.
 
+## Plugin developers
+
+Plugin developers willing to support JCasC in their plugin should [check out our guidelines](docs/PLUGINS.md).
```
Comment (timja) [inline README.md:249]: I think this should be:

```suggestion
Plugin developers willing to support JCasC in their plugin should [check out our guidelines](docs/CONTRIBUTING.md).
```

Comment (timja) [reply README.md:249]: ah i see what you mean

Comment (timja) [reply README.md:249]: ah i see what you mean

**Comment by alb-i986** [comment]
Haha ya sorry, I'm talking about other plugins' developers.
May be a bit ambigous, true.

File: README.md [hunk 2]
```diff
@@ -244,6 +244,10 @@ Kubernetes users:\
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=18341) for known compatibility issues.
 
+## Plugin developers
```
Comment (alb-i986) [inline README.md:247]: Should I change the title to "Other plugins' developers"? Or "Third party plugin developers", or whatever you think it's best.

File: README.md (hunk 1 above)
Comment (timja) [reply README.md:249]:
```suggestion
Plugin developers wanting to support JCasC in their plugin should [check out our how-to guide](docs/PLUGINS.md).
```

File: README.md (hunk 2 above)
Comment (timja) [reply README.md:247]: what about?
```suggestion
## Adding JCasC support to a plugin
```

**Pull request merged**